### PR TITLE
Validate redirects

### PR DESCRIPTION
**Pull request by andrewgarner**

### CODE REVIEW

**Comment by gjvis** [comment]
How does this get run?

**Comment by andrewgarner** [comment]
At the moment, manually.

Regards,

Andrew Garner

Guidance Guarantee Programme
HM Treasury

+44 7971 232 140

File: redirects.rb [hunk 1]
```diff
@@ -0,0 +1,72 @@
+require 'bundler/setup'
+require 'faraday'
+
+def test_redirect(url, location, status = 301)
```
Comment (stevenwilkin) [inline redirects.rb]: Are we going to have anything other than 301 redirects?

File: redirects.rb [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+  if actual_status != status
+    throw "Expected #{status}, got #{actual_status}"
+  end
+
+  if actual_location != location
+    throw "Expected #{location}, got #{actual_location}"
+  end
+
+  true
```
Comment (stevenwilkin) [inline redirects.rb]: Any need to return `true` ?

**Comment by stevenwilkin** [comment]
Are we abandoning this?

**Comment by andrewgarner** [comment]
I think we should yeh. We're monitoring this elsewhere now.

**Pull request closed without merging**